### DISCUSSION TITLE
How do we add ignores to the linter?

### DISCUSSION THREAD

**Discussion by scpedicini** [lukehaas/RunJS#601] [Q&A]
I'd like to keep typescript checking/linting on if possible, but there are certain "errors" that it flags that I want to turn off such as no Used locals. I tried editing the `~/Library/Application Support/runjs/project/tsconfig.json` and adding `noUnusedLocals` set to false but it didn't appear to make a difference even after restarting the app.

The errors about wanting to convert require statements to ignores, and then when you do, you get warnings about d.ts files missing are also kind of distracting, this is another set that I'd like to be able to disable.

**Comment by lukehaas** [comment]
There isn't currently a way to configure this, but if you and others want it, I will consider adding options.

You can avoid the d.ts warnings by installing types via NPM.
